src/main/java/com/bulkbuy/enterprise/dto/Order.java

```diff
@@ -1,23 +1,98 @@
 package com.bulkbuy.enterprise.dto;
 
 import lombok.Data;
+
+import javax.persistence.*;
 import java.util.Date;
 
 /**
  * DTO, handles data for orders.
  */
-public @Data class Order {
-
+@Entity
+@Table(name = "order")
+public class Order {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private int orderId;
```

Comment:
Here you have a Order class with multiple attributes. You also have constructors. The Order class is annotated with @ Entity, indicating that it is a JPA entity. (Because no @ Table annotation exists, it is assumed that this entity is mapped to a table named Order.)
The Order object’s id property is annotated with @ Id so that JPA recognizes it as the object’s ID. The id property is also annotated with @ GeneratedValue to indicate that the ID should be generated automatically. 
Sources: https://www.baeldung.com/jpa-entities